mlflow/recipes/resources/agent_evaluation_template.html

```diff
@@ -0,0 +1,261 @@
+    }
+    .tab:hover {
+      color: rgb(14, 83, 139) !important;
+    }
+    .tab.active {
+      border-bottom: 3px solid rgb(34, 114, 180) !important;
+    }
+    .link {
+      margin-left: 12px;
+      display: inline-block;
+      text-decoration: none;
+      color: rgb(34, 114, 180) !important;
+      font-size: 13px;
+      font-weight: 400;
+    }
+    .link:hover {
+      color: rgb(14, 83, 139) !important;
+    }
+    .link-content {
+      display: flex;
+      gap: 6px;
+      align-items: center;
+    }
+  </style>
+</head>
+<body>
+  <div style="display: flex; align-items: center">
+    The logged model is compatible with the Mosaic AI Agent Framework.
+    <button onclick="showCode()">
+      See how to evaluate the model&nbsp;
+      <span
+        role="img"
+        aria-hidden="true"
+        class="anticon css-6xix1i"
+        style="font-size: 14px"
+        ><svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="1em"
+          height="1em"
+          fill="none"
+          viewBox="0 0 16 16"
+          aria-hidden="true"
+          focusable="false"
+          class=""
+        >
+          <path
+            fill="currentColor"
+            fill-rule="evenodd"
+            d="M8 8.917 10.947 6 12 7.042 8 11 4 7.042 5.053 6z"
+            clip-rule="evenodd"
+          ></path>
+        </svg>
+      </span>
+    </button>
+  </div>
+  <div id="code" style="display: none" onclick="showCode()">
+    <h1>
+      Agent evaluation
+      <a
+        class="link"
+        href="https://docs.databricks.com/en/generative-ai/agent-evaluation/synthesize-evaluation-set.html?utm_source=mlflow.log_model&utm_medium=notebook"
+        target="_blank"
+      >
+        <span class="link-content">
+          Learn more
+          <span role="img" aria-hidden="true" class="anticon css-6xix1i"
+            ><svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="1em"
+              height="1em"
+              fill="none"
+              viewBox="0 0 16 16"
+              aria-hidden="true"
+              focusable="false"
+              class=""
+            >
+              <path
+                fill="currentColor"
+                d="M10 1h5v5h-1.5V3.56L8.53 8.53 7.47 7.47l4.97-4.97H10z"
+              ></path>
+              <path
+                fill="currentColor"
+                d="M1 2.75A.75.75 0 0 1 1.75 2H8v1.5H2.5v10h10V8H14v6.25a.75.75 0 0 1-.75.75H1.75a.75.75 0 0 1-.75-.75z"
+              ></path></svg></span></span
+      ></a>
+    </h1>
+    <p class="info">
+      Copy the following code snippet in a notebook cell (right click → copy)
+    </p>
+    <div class="tabs">
+      <div class="tab active" onclick="tabClicked(0)">Using synthetic data</div>
+      <div class="tab" onclick="tabClicked(1)">Using your own dataset</div>
+    </div>
+    <div style="height: 400px">
+      <pre
+        class="active"
+      ><code class="language-python">%pip install -qqq -U databricks-agents
```

Comment:
do we need to run `%restart_python` after `%pip install?`

Reply:
We actually don't need it for the databricks-agents package -- tested in a notebook

Reply:
I got this error:

<img width="936" alt="image" src="https://github.com/user-attachments/assets/a16593cc-6cda-4e69-9b41-0f414ca62610" />

Reply:
ah nice catch! I couldn't repro it, but then i realized serverless is probably caching site-packages, and detach & reattach was not enough, I had to "reset" the state. Though it revealed another issue... I'll have to encourage users to run the pip install + restartLibrary() in a separate cell.

![CleanShot 2024-12-16 at 23 24 21@2x](https://github.com/user-attachments/assets/88cc6e9d-99f8-4c3a-a4a5-40bc7b8502d8)

Reply:
Added a comment to encourage the user to run the first 2 lines in a separate cell